services/auth/token/helpers/generateIAMPolicy.js

```diff
@@ -0,0 +1,21 @@
+// Helper function to generate an IAM policy
+const generateIAMPolicy = function(principalId, effect, resource) {
```

Comment:
function(...
function (...
According to my linter we're missing space

Reply:
This should be an arrow function, for the sake of it